src/version/version_test.go

```diff
@@ -0,0 +1,146 @@
+package version
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/NeowayLabs/semantic-release/src/tests"
+)
+
+const (
+	shouldbeEqualMessage = "Values should be equal:\n Actual: %v\nExpected: %v"
+)
+
+type RepositoryVersionControlMock struct {
+	message string
+	err     error
+}
+
+func (r *RepositoryVersionControlMock) GetMessage() (string, error) {
+	return r.message, r.err
+}
+
+func (r *RepositoryVersionControlMock) GetVersionUpdateType(message string) (string, error) {
+	return message, r.err
+}
+
+func (r *RepositoryVersionControlMock) UpgradeRemoteRepository() error {
+	return r.err
+}
+
+type FilesVersionControlMock struct {
+	err error
+}
+
+func (f *FilesVersionControlMock) UpgradeChangeLog(path string) error {
+	return f.err
+}
+func (f *FilesVersionControlMock) UpgradeVariableInFiles(variableName string, filesList []string) error {
+	return f.err
+}
+
+type fixture struct {
+	repoVersionMock  *RepositoryVersionControlMock
+	filesVersionMock *FilesVersionControlMock
+}
+
+func setup() *fixture {
+	return &fixture{repoVersionMock: &RepositoryVersionControlMock{}, filesVersionMock: &FilesVersionControlMock{}}
+}
+
+func TestGetMessageError(t *testing.T) {
+	f := setup()
+	f.repoVersionMock.err = errors.New("Something went wrong while getting message.")
+	versionService := NewService(f.repoVersionMock, f.filesVersionMock)
+	_, actualErr := versionService.repoVersionControl.GetMessage()
```

Comment:
why are you accessing one private variable? @esequielvirtuoso

Reply:
some for all code where you access one private variable @esequielvirtuoso

Reply:
Which variable?

Are you talking about `repoVersionControl`?

Reply:
Done.